packages/api/src/sequelize/migrations/2023-12-12_00_create-patient-discovery.ts

```diff
@@ -0,0 +1,43 @@
+import { DataTypes } from "sequelize";
+import type { Migration } from "..";
+import * as shared from "../migrations-shared";
+
+const tableName = "patient_discovery_result";
```

Comment:
Do we know how we'll query this table? We can add indexes later, but if we know that we'll query for created+status or just created, could create the index out of the bat and maybe save a DB migration later.

NABD otherwise.

Reply:
I see it as a query for all results in a certain time period for a given patient

Reply:
I see we're not setting the ID as PK but also not creating any indexes. If the query you mentioned is the one we're going to use, maybe we define this index now? Or can you ticket both the index and the sweeper logic just so we don't forget as part of the go live, please? Even if it comes a couple weeks afterwards, it's defs something we'll need sooner than later, as getting to 1MM records on this table won't take long.